web-components/src/internal-components/color-table/ColorTable.ts

```diff
@@ -1,59 +1,61 @@
 import "@/components/table-advanced/TableAdvanced";
-import { customElement, html, internalProperty, LitElement, PropertyValues } from "lit-element";
-import { colorTableData } from "@/wc_scss/colors/vars/color-table-data-string";
 import { TableAdvanced } from "@/components/table-advanced/TableAdvanced";
+import { customElementWithCheck } from "@/mixins";
+import { colorTableData } from "@/wc_scss/colors/vars/color-table-data-string";
+import reset from "@/wc_scss/reset.scss";
+import { html, internalProperty, LitElement, PropertyValues } from "lit-element";
 import { nothing } from "lit-html";
 import { colorCellTemplates } from "./data";
-import { allColorTemplates } from "./utils";
-import reset from "@/wc_scss/reset.scss";
 import styles from "./scss/module.scss";
+import { allColorTemplates } from "./utils";
 
-@customElement("color-table")
-export class ColorTableSandbox extends LitElement {
-  @internalProperty() colorTokenTableData:
-    | { config: TableAdvanced.Config; data: TableAdvanced.Data }
-    | undefined = undefined;
+export namespace ColorTableSandbox {
+  @customElementWithCheck("color-table")
```

Comment:
this is actually not needed :) this is an internal component and will never be exported from this library, so it does not need to have this custom decorator